### DISCUSSION TITLE
point_0704

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#259] [Roy]
"20200527T030549_20200527T031643_T49QDC"
![image](https://user-images.githubusercontent.com/54947195/147532255-42330d5e-1f3d-4e1a-9dba-0b32d0c98242.png)

"20190428T030551_20190428T032153_T49QDC"
![image](https://user-images.githubusercontent.com/54947195/147532447-890576be-39f2-47d3-b669-a11eac338b51.png)
![image](https://user-images.githubusercontent.com/54947195/147532490-92aef198-63b6-41c2-b6eb-b019cab6beaf.png)

"20190612T030549_20190612T032055_T49QDC"
![image](https://user-images.githubusercontent.com/54947195/147532663-9c5cc555-b090-48ca-b636-527a8cf6bb49.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20190612T030549_20190612T032055_T49QDC"
![Captura](https://user-images.githubusercontent.com/86502862/153085334-1cbd4b3d-7aad-4af1-a0c6-7d5bcdbc5526.JPG)
